tests/trait_mapping/test_output.py

```diff
@@ -43,37 +43,47 @@ def test_output_trait_mapping(self):
 
 class TestGetMappingsForCuration(unittest.TestCase):
     def test_get_mappings_for_curation(self):
```

Comment:
This method looks like 3 different test cases to me (each of them explicitly documented), so it would be more clear if split in 3 test methods.

Reply:
This is a very good point. Done.